### ISSUE TITLE
Unit Tests for Image Uploading/Deleting

### ISSUE THREAD

**Issue by austintoddj** [austintoddj/Canvas#50]
PHPUnit tests should be written to cover the applications' ability to create and delete a file/image.

**Comment by RDelorier** [comment]
Looks like I'll have some time this weekend to help out with some stuff. I think testing is more pressing than cleaning up the profile pages so I'll work on these instead. If you want to line up some more "needs testing issues" for me I'll take them this weekend. You may want to require tests for contributions in the future to help protect the project.

**Comment by austintoddj** [comment]
@RDelorier That sounds great! I have no problem at all writing up some stuff that could help protect the integrity of the app, perhaps some contribution guidelines and such, just don't know how to word it. What kind of tests do you think would be applicable in this case, aside from writing unit tests making sure all of the features of the app actually function?

**Comment by RDelorier** [comment]
Just making everything works is definitely a good start. Bug reports will help with coming up with tests for edge cases (like that msql packet issue last week).

**Comment by austintoddj** [comment]
Yup good thought. Ok I'll try to hammer out some more testing issues and get them up before the end of the day.

**Comment by RDelorier** [comment]
Since I don't know the app very well would you mind mentioning where the feature is handled at (ie: FileUploadController). I didn't even realize there were file uploads until I saw this issue.

**Comment by austintoddj** [comment]
No problem.

**Comment by austintoddj** [comment]
@RDelorier How does the updated issue on this one look for you?

**Comment by RDelorier** [comment]
Wow that's perfect, don't make too easy for me 😂😂
